ontologytimemachine/utils/config.py

```diff
@@ -90,30 +99,33 @@ def enum_parser(enum_class: Type[E], value: str) -> E:
 
 def parse_arguments(config_str: str = "") -> Config:
     default_cfg: Config = Config()
-    parser = argparse.ArgumentParser(description="Process ontology format and version.")
+    parser = argparse.ArgumentParser(
+        description="Process ontology format and version.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
```

Comment:
I also added this here, but it didn't add the default values to the help and the trick I did is also from the link you gave me :)